### ISSUE TITLE
make pulp init interactive?

### ISSUE THREAD

**Issue by garyb** [purescript-contrib/pulp#78]
Currently the `bower.json` generated is less than ideal as it's missing a few fields that probably should be there: `license`, and more importantly, `repository`. Without the `repository` field purescript libraries cannot be published to pursuit, and it's an easy thing to forget.

Perhaps we should use an interactive init like `npm` and `bower` so these fields along with other common things like `description`, `homepage`, etc. can be set?

**Comment by ghost** [comment]
If possible, my preference would be to have these things checked as a sort of linting process prior to publishing. IIRC, this is how `cargo` for Rust works. Often times when I start a little project I don't want to be bothered with filling all of that stuff in and I find the interactive approach a bit bothersome.

If it is changed to be interactive though, it would be nice if there were a way to easily disable it (with a short flag) or allow the user to skip everything at the first question.

**Comment by tfausak** [comment]
I would prefer keeping it non-interactive but allowing different project templates, like [stack-templates](https://github.com/commercialhaskell/stack-templates). That way you could do `pulp init --template simple` for little projects and `pulp init --template kitchen-sink` for big ones.

**Comment by hdgarrood** [comment]
I agree with @freebroccolo (would rather have a linting process). We do have `psc-publish --dry-run` now, which improves the situation quite a lot, I think. There's also #57, which I really will get around to soon...
